app/league/slack_messenger.py

```diff
@@ -28,6 +26,9 @@ def init_app(self, app):
 
     def update_configuration(self, config):
         """Update Slack Messenger configuration."""
+        assert 'enabled' in config
+        if type(config['enabled']) == str:
```

Comment:
I'll raise an issue to discuss this (after dinner). I like the idea of serialization, but it's a bit antithetical to SQL, isn't it?